native/packages/react-native-bpk-component-text/src/BpkText-test.common.js

```diff
@@ -80,6 +80,34 @@ const commonTests = () => {
         expect(tree).toMatchSnapshot();
       });
     });
+    TEXT_STYLES.forEach(textStyle => {
+      it(`should render correctly with textStyle="${textStyle} and weight="emphasized"`, () => {
+        const tree = renderer
+          .create(
+            <BpkText weight={WEIGHT_STYLES.emphasized} textStyle={textStyle}>
+              Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean
+              commodo ligula eget dolor. Aenean massa. Cum sociis natoque
+              penatibus et magnis dis parturient montes, nascetur ridiculus mus.
+            </BpkText>,
+          )
+          .toJSON();
+        expect(tree).toMatchSnapshot();
+      });
+    });
+    TEXT_STYLES.forEach(textStyle => {
```

Comment:
Shouldn't this only tests the variants that support heavy?

Reply:
See commit `Remove tests for invalid emphasize/heavy combinations`